runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -116,6 +117,11 @@ static int getCoresPerLocale(int nomultithread) {
     chpl_warning("CHPL_LAUNCHER_CORES_PER_LOCALE must be > 0.", 0, 0);
   }
 
+  if (cpusPerTaskString) {
+    numCores = atoi(cpusPerTaskString);
```

Comment:
Code thriftiness: since `numCores` is local, could just do `return atoi(cpusPerTaskString);`?

Reply:
Eh, out of laziness I'll probably just leave it unless you object. (I just copied from the `CHPL_LAUNCHER_CORES_PER_LOCALE` path above)

Reply:
No problem, and it does echo other coding nearby.